pegasus/sites.v3/code.org/views/music/music_lab_additional_resources.haml

```diff
@@ -0,0 +1,37 @@
+:ruby
+  additional_resources = [
+    {
+      title: hoc_s('music_lab.additional_resources.catalog.title'),
+      image: "/images/action-blocks/action-block-catalog.png",
+      desc: hoc_s('music_lab.additional_resources.catalog.desc'),
+      url: CDO.studio_url("/catalog"),
+      button_label: hoc_s('music_lab.additional_resources.catalog.button'),
+    },
+    {
+      title: hoc_s('music_lab.additional_resources.videos.title'),
+      image: "/images/action-blocks/action-block-videos.png",
+      desc: hoc_s('music_lab.additional_resources.videos.desc'),
+      url: "/videos",
+      button_label: hoc_s('music_lab.additional_resources.videos.button'),
+    },
+    {
+      title: hoc_s('music_lab.additional_resources.hoc.title'),
+      image: "/images/action-blocks/action-block-hoc.png",
+      desc: hoc_s('music_lab.additional_resources.hoc.desc'),
+      url: "/hourofcode",
+      button_label: hoc_s('music_lab.additional_resources.hoc.button'),
+    },
+  ]
+
+.action-block__wrapper.action-block__wrapper--three-col
+  - additional_resources.each do |additional_resource|
+    .action-block.action-block--one-col.flex-space-between.white
+      .content-wrapper
+        %h3
+          = additional_resource[:title]
+        %img{src: "#{additional_resource[:image]}", alt: ""}
```

Comment:
out of curiousity, why can't this be `src: additional_resources[:image]`

Reply:
Ah good catch, just updated 👍